build.sbt

```diff
@@ -47,7 +47,8 @@ val installAll =
      |export COMPOSER_HOME=/opt/composer &&
      |mkdir -p $$COMPOSER_HOME &&
      |apk update &&
-     |apk add bash curl git php php-xml php-cli php-pdo php-curl php-json php-phar php-ctype php-openssl php-dom &&
+     |apk add bash curl git php5 php5-xml php5-cli php5-pdo php5-curl php5-json php5-phar php5-ctype php5-openssl php5-dom &&
+     |ln -s /usr/bin/php5 /usr/bin/php &&
      |curl -sS https://getcomposer.org/installer | php -- --install-dir=/bin --filename=composer &&
```

Comment:
If you installed composer with php 5 you would not need to make this kind of aliases

Reply:
yeah I tried that, but then had problems when the docker runs the phpcpd, so instead of having there "php5" while generating the command to run, I thought it would be better just to always have a "php"